tools/maven/checkstyle.xml

```diff
@@ -224,10 +224,6 @@ This file is based on the checkstyle file of Apache Beam.
 			<property name="illegalPkgs" value="io.netty"/>
 			<message key="import.illegal" value="{0}; Use flink-shaded-netty instead."/>
 		</module>
-		<module name="IllegalImport">
-			<property name="illegalPkgs" value="com.google.common"/>
```

Comment:
should we put here `org.apache.flink.shaded` to make it failing once flink-shaded is added to import?